### DISCUSSION TITLE
Get number of stars and forks a repository has.

### DISCUSSION THREAD

**Discussion by albas99** [community/community#31111] [Apps, API and Webhooks] [answered]
This is something I think should be available as part of the repositories endpoint. To be able to see how many stars and forks a repo has, especially public/open-source repos.

**Comment by ichenglin** [comment, marked as answer]
There is already a method provided by Github for retrieving repository information,
which is known as the **[Github Rest API](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api)**.

There's **tons of features** provided, however **in this case** you want the **[get repository](https://docs.github.com/en/rest/repos/repos#get-a-repository)** endpoint for repository information.
```https://api.github.com/repos/{owner}/{repository}```

For example, if you want to know **how many stars or forks** the **[community/community](https://github.com/community/community)** repository has,
you could **simply fetch** from **`https://api.github.com/repos/community/community`** for such information.

![image](https://user-images.githubusercontent.com/41904540/187069812-2046f97a-0b3a-4e42-8cf5-df266accbb8a.png)
